db/migrate/20230607203415_create_likes.rb

```diff
@@ -0,0 +1,5 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes, &:timestamps
+  end
+end
```

Comment:
- [ ] Kindly set up references for the author and post.

```ruby
create_table :likes do |t|
    t.timestamps 
    t.references :author, foreign_key: { to_table: 'users' } 
    t.references :post, foreign_key: true 
  end
```

Reply:
let me work on that